### ISSUE TITLE
Error with python 3.7

### ISSUE THREAD

**Issue by slandau3** [jhartz/gradefast#2]
When I try to use gradefast with python 3.7 I get the following error:
```
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/slandau/Desktop/school/sli/gradefast/gradefast/__main__.py", line 23, in <module>
    from gradefast.config.local import GradeFastLocalModule
  File "/Users/slandau/Desktop/school/sli/gradefast/gradefast/config/local.py", line 20, in <module>
    class GradeFastLocalModule(Module):
  File "/Users/slandau/Desktop/school/sli/gradefast/gradefast/config/local.py", line 67, in GradeFastLocalModule
    def provide_local_host(self) -> InjectableClass[hosts.LocalHost]:
  File "/Users/slandau/Desktop/school/sli/gradefast/external/pyprovide/pyprovide.py", line 521, in handle
    return_type = _get_provider_return_type(provider_method, "Class provider")  # type: Any
  File "/Users/slandau/Desktop/school/sli/gradefast/external/pyprovide/pyprovide.py", line 467, in _get_provider_return_type
    (provider_type, provider_method.__name__, return_type))
pyprovide.BadProviderError: Class provider "provide_local_host" return type annotation "typing.Type[gradefast.hosts.LocalHost]" is not a type
```
I believe the problem is with the pyprovide extension.

A temporary workaround for this is running the script in a virtual environment running python 3.5

**Comment by jhartz** [comment]
Looks like this is a PyProvide issue: https://github.com/jhartz/pyprovide/issues/1

**Comment by jhartz** [comment]
Fixed in: jhartz/pyprovide#2

GradeFast submodule updated in: https://github.com/jhartz/gradefast/commit/392407720326d2302efeb7d93f3791dbe6ae8c97
